cr-examples/triton/src/test/java/oracle/code/triton/TestAddKernel.java

```diff
@@ -92,12 +93,12 @@ static void add_kernel(Ptr x_ptr,  // *Pointer* to first input vector.
     @TritonTestExtension.Kernel("add_kernel")
     @Test
     public void test(TritonTestData t) {
-        List<Type> argTypes = List.of(
-                new PtrType(float.class),
-                new PtrType(float.class),
-                new PtrType(float.class),
-                int.class,
-                new ConstantType(int.class, 64));
+        List<TypeElement> argTypes = List.of(
```

Comment:
I think this is still syntactically pleasing, but conceptually more straightforward. Good job!